Cron/Order.php

```diff
@@ -79,8 +88,10 @@ public function __construct(
      */
     public function execute() : void
     {
+        $this->config->initSdkConfiguration();
```

Comment:
@cwalv Thanks for catching this! I had this on my LightSail and didn't realize it wasn't in my commits. We may want to move this to before we declare the LoggingClient on line 81. That way the logging client object is aware of the config values that have been initialized.

Reply:
good catch; fixed